protocol/validation/vmcontext.go

```diff
@@ -97,13 +101,17 @@ func NewTxVMContext(vs *validationState, entry bc.Entry, prog *bc.Program, args
 	return result
 }
 
-func witnessProgram(prog []byte) []byte {
+func witnessProgram(prog []byte, lockedAssetID bc.AssetID, args [][]byte) []byte {
```

Comment:
https://go-critic.github.io/overview.html#ifElseChain-ref

avoiding if-else-chain here can help improve the readability

Reply:
ok